manifests/filter/open_tracing.pp

```diff
@@ -0,0 +1,182 @@
+# == Resource:  repose::filter::open_tracing
+#
+# This is a resource for enabling emitting traces to an OpenTracing tracer (current supports Jaeger).
+# Only usable in >= 8.8.3.0
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename.
+# Defaults to <tt>open-tracing.cfg.xml</tt>
+#
+# [*service_name*]
+# String. Application name
+# Defaults to <tt>repose</tt>
+#
+# [*connection_type*]
+# String that sets up opentracing connection.  Can be either http or udp
+# Defaults to <tt>udp</tt>
+#
+# [*connection_endpoint*]
+# String. Set to http endpoint to push traces to.  Required if using http connection.
+# Defaults to <tt>http://localhost:12682/api/traces</tt>
```

Comment:
Is this the right choice for the default? i thought you figured out that the client was automatically appending the path?

Reply:
it appends `?format=jaeger.thrift` but not the path :(

Reply:
for reference: https://github.com/jaegertracing/jaeger-client-java/blob/2fa8f3ed17e88114f6cd7114747ed298933dabd8/jaeger-core/src/main/java/io/jaegertracing/senders/HttpSender.java?utf8=%E2%9C%93#L89-L90

Reply:
second question is that a reasonable default, or is that just what yours was while you were testing?

Reply:
If you are to spin up docker jaeger all-in-one-tracing, it's a reasonable default.  It would need to change to point to a real tracer.  Also, it maps to the default port specified in https://www.jaegertracing.io/docs/deployment/#collectors

Reply:
so is the expected deployment from users to use the docker image? if not defaulting to that isn't the right choice.

Reply:
That or some local setup.  I'm not sure what other defaults we can use if not that.